meetings/2024-04-11.md

```diff
@@ -3,14 +3,18 @@
 ## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
 
 ### Attending
-- 
+- Jason Laura 
+- Victor Silva 
+- Kelvin Rodriguez 
+- Audrie Fennema
+- Adam Paquette
 
 ## Agenda / Notes
 - Astro Conda-forge recipes and maintainers (Andrew)
 - External Contributors (Kris)
 
 ## Discussions for next meeting
-- 
+- The need for an ISIS TC meeting (Kelvin)
```

Comment:
@AndrewAnnex @KrisBecker @jlaura @acpaquette 

Pinging member that are frequent contributors, but I think since we have had a few meetings in the row that have been unproductive, we should consider whether we need to continue doing a monthly meeting going forward.

Reply:
Some more seed for discussion: 

1. Are we doing enough to advertise these? Should we put out a bulletin to the public? 
2. Should we expand the purpose of the TC? For example, make it more of a public forum on ISIS and the Astro software suite and not strictly limited to ISIS governance discussion. 
3. Consider new methods to encourage discussion outside of in person meetings